app/src/main/java/cm/aptoide/pt/view/FragmentModule.java

```diff
@@ -314,4 +317,17 @@ public FragmentModule(Fragment fragment, Bundle savedInstance, Bundle arguments,
     return new BundleEvent(arguments.getString(BundleCons.TITLE),
         arguments.getString(BundleCons.ACTION));
   }
+
+  @FragmentScope @Provides WizardPresenter providesWizardPresenter(
+      AptoideAccountManager aptoideAccountManager, CrashReport crashReport,
+      AccountAnalytics accountAnalytics) {
+    Integer[] transitionColors = {
+        fragment.getContext()
+            .getResources().getColor(R.color.wizard_color_1_blue), fragment.getContext()
+        .getResources().getColor(R.color.wizard_color_2_green), fragment.getContext()
+        .getResources().getColor(R.color.wizard_color_3_orange)
+    };
+    return new WizardPresenter((WizardView) fragment, aptoideAccountManager, crashReport,
+        accountAnalytics, transitionColors);
```

Comment:
The presenter should not know anything about how view shows view details like colour transitions.